### ISSUE TITLE
Remove Fontawesome

### ISSUE THREAD

**Issue by artitudinale1** [FreeCodeCampLondon/FCCLND#43]
We use font awesome just for twitter / facebook icon in main pages and icons on campers page.

Create a sprite with all icons used (you can download the official one from web) 

Resource **https://css-tricks.com/css-sprites/**

Use CSS  `backgound-position` to display icons

**Comment by GarySiu** [comment]
Any objections to using [icomoon](https://icomoon.io) to generate a custom icon font? Sprites are neat but if we use PNG then we'd need to create an image for every colour and size you want. I found that we're actually using 5 icons and 2 of which we use in a different size and colour. This doesn't include hover states. SVGs are slightly more flexible but they aren't as well supported and only let you style 2 colours with CSS at the moment. I found all the icons needed in icomoon's free set and edited the FCC logo for import. Also this gives us more room for extension.
